.github/workflows/build.yml

```diff
@@ -90,9 +90,34 @@ jobs:
           override: true
 
       - name: Generate Code Coverage
-        uses: actions-rs/tarpaulin@v0.1
+        uses: actions-rs/tarpaulin@v0.1.3
```

Comment:
This change is not really needed. I thing `@0.1` should automatically pick up patch versions.

Reply:
That was to tickle the CI to see if a rerun would make the code coverage work.